python/zensical/config.py

```diff
@@ -448,6 +448,8 @@ def _apply_defaults(config: dict, path: str) -> dict:
     for fence in superfences.get("custom_fences", []):
         if isinstance(fence.get("format"), str):
             fence["format"] = _resolve(fence.get("format"))
+            if isinstance(fence.get("validator"), str):
```

Comment:
So a validator can only be present if a format is present, if I get the indentation correct?